UPGRADE.md

```diff
@@ -0,0 +1,44 @@
+Netgen Layouts Remote Media upgrade instructions
+===============================================
+
+Upgrade from 1.0 to 2.0
+-----------------------
+
+### Netgen Remote Media upgrade
+
+Version 2 of this bundle uses Netgen Remote Media version 3 which differs a lot than version 2, which is being used by version 1 of this bundle so first read [Netgen Remote Media upgrade instructions](https://github.com/netgen/NetgenRemoteMediaBundle/blob/NGSTACK-533-local-resource-storage/docs/UPGRADE.md#upgrade-from-20-to-30)!
```

Comment:
Link needs to be fixed, presumably when NGRM v3 is merged to master.

Reply:
@emodric NGRM master is currently for 2.0 and will probably remain like this for a while, but I've changed from specific branch to `3.0` branch, because this NGSTACK-533 has been merged into 3.0. Fixed in https://github.com/netgen-layouts/layouts-remote-media/pull/15/commits/f723fdc25f3cf0ea2c4c710e96521a1b3572adff